src/cpp/tests/testthat/run-tests.R

```diff
@@ -0,0 +1,18 @@
+library(testthat)
+
+runAllTests <- function(sourceDir, outputDir, filter = NA)
+{
+   testThatDir <- file.path(sourceDir, "tests/testthat")
+   
+   if (!is.na(filter))
+   {
+      tests <- testthat::test_dir(testThatDir, filter = filter)
+   } else
```

Comment:
nit: else should be on its own line

Reply:
For some reason the IDE was giving me syntax errors if I put the else on its own line. Probably a false positive

Reply:
If you can reproduce this (diagnostics system highlighting an error in otherwise valid R code) can you file an issue?

Reply:
It seems to be gone now.